skimage/future/graph/rag.py

```diff
@@ -264,6 +272,30 @@ def _add_node_silent(self, n):
         .. seealso:: :func:`networkx.Graph.add_node`."""
         super(RAG, self).add_node(n)
 
+    def nodes_iter(self, *args, **kwargs):
```

Comment:
@jni - do you mean - I should just replace the calls to `nodes_iter` and `edges_iter` with calls to `nodes` and `edges`?  I could do that, for sure; I didn't do it because I was worried that the graph might be large enough that, using old networkx, a call to `graph.nodes()` would generate a very big list.